interview-questions.md

```diff
@@ -7,44 +7,72 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
-
-  Researched answer:
-
-
+    - Your answer:
+      > To add a foreign key you would need to run a change migration. The foreign key would be named cohort_id and be on the student model. I've had to implement this process while creating an api that tracked animal sightings, I had accidentaly forgot to add the foreign key but implimented a change migration to ensure all animals could have many sightings.
+    - Researched answer:
+      > If you forget to add a foreign key or want to modify one, you will need to generate a migration to modify the schema so that you don't break your database. In this situation we would want to generate a migration named AddForeignKeyToStudents. The migration would be typed using the camel cased naming convention and be close to what you're trying to do (i.e. AddForeignKeyToStudents). The reason the change would be on the Student model is because the Cohort can have many students, but the individual students only have one cohort, so the pointer (foreign key) goes in the table that has records that belong_to another model record.
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
-
-  Researched answer:
+    - Your answer:
+      > Out of the eight RESTful routes the show, edit, update, delete, and destroy all need params since they all require an id to be present to work on a specific record.
```

Comment:
You are serious about deleting. I see that you meant show, edit, update, and destroy